packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/templates/defaultSearchBoxInput.html

```diff
@@ -33,6 +32,7 @@
                 aria-haspopup="listbox"
                 aria-autocomplete="list"
                 data-cy="search-box-input"
+                data-value={inputValue}
```

Comment:
I'm adding this to make sure the component renders when the value of this public property changes, 

I just discovered this in LWC, if a public property is not used in the HTML or in a getter, the component will not be reactive to it.

Reply:
Yea that's why they had the @track before.

Reply:
But  @track is not meant to be used with @api